### PR TITLE
Update replication docs for InfluxDB 2.4

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 1]
```diff
@@ -50,33 +58,26 @@ Use InfluxDB replication streams (InfluxDB Edge Data Replication) to replicate t
...
+    - {{% oss-only %}}Remote bucket name or ID to replicate writes to. If replicating to **InfluxDB Enterprise**, use the `db-name/rp-name` bucket name syntax.{{% /oss-only %}}
+    - {{% cloud-only %}}Remote connection ID{{% /cloud-only %}}
+    - {{% cloud-only %}}InfluxDB OSS bucket ID to replicate writes from{{% /cloud-only %}}
+    - {{% cloud-only %}}InfluxDB Cloud bucket ID to replicate writes to{{% /cloud-only %}}
 
-    {{% /cloud-only %}}
 
     ```sh
     influx replication create \
       --name example-replication-stream-name \
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:77]:
```suggestion
      --name REPLICATION_STREAM_NAME \
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 2]
```diff
@@ -50,33 +58,26 @@ Use InfluxDB replication streams (InfluxDB Edge Data Replication) to replicate t
...
       --remote-id 00xoXXXo0X0x \
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:78]:
```suggestion
      --remote-id REPLICATION_REMOTE_ID \
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 3]
```diff
@@ -50,33 +58,26 @@ Use InfluxDB replication streams (InfluxDB Edge Data Replication) to replicate t
...
       --local-bucket-id Xxxo00Xx000o \
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:79]:
```suggestion
      --local-bucket-id INFLUX_BUCKET_ID \
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 4]
```diff
@@ -50,33 +58,26 @@ Use InfluxDB replication streams (InfluxDB Edge Data Replication) to replicate t
...
-      --remote-bucket-id 0xXXx00oooXx
+      --remote-bucket example-remote-bucket
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:80]:
```suggestion
      --remote-bucket REMOTE_INFLUX_BUCKET_NAME_OR_ID
```

**Comment by sanderson** [comment]
@samhld Thoughts on adding an Enterprise-specific version of this doc to the Enterprise documentation?

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 5]
```diff
@@ -85,6 +86,161 @@ InfluxDB {{% cloud-only %}}Cloud {{% /cloud-only %}}bucket.
...
+<!---------------------------------- END CLI ---------------------------------->
+
+{{% /tab-content %}}
+{{% tab-content %}}
+
+<!--------------------------------- BEGIN API --------------------------------->
+
+1.  In your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS instance, use
+    the `/api/v2/remotes` endpoint with the `POST` request method to create a
+    remote connection to replicate data to.
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:98]:
```suggestion
1.  Send a `POST` request to your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS  `/api/v2/remotes` endpoint to create a remote connection to replicate data to.
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 6]
```diff
@@ -85,6 +86,161 @@ InfluxDB {{% cloud-only %}}Cloud {{% /cloud-only %}}bucket.
...
+
+    {{< keep-url >}}
+    ```sh
+    curl --request GET \
+      http://localhost:8086/api/v2/remotes?orgID=INFLUX_OSS_ORG_ID \
+      --header 'Authorization: Token INFLUX_OSS_TOKEN' \
+    ```
+
+2.  In your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS instance, use the
+    `/api/v2/replications` endpoint to create a replication stream.
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:164]:
```suggestion
2.  Send a `POST` request to your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS 
    `/api/v2/replications` endpoint to create a replication stream.
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 7]
```diff
@@ -85,6 +86,161 @@ InfluxDB {{% cloud-only %}}Cloud {{% /cloud-only %}}bucket.
...
+    "maxQueueSizeBytes": 67108860,
+    "name": "Example replication stream name",
+    "orgID": "INFLUX_OSS_ORG_ID",
+    "remoteBucketName": "REMOTE_INFLUX_BUCKET_NAME",
+    "remoteID": "REMOTE_ID",
+  }'
+```
+
+Once a replication stream is created, InfluxDB {{% oss-only %}}OSS{{% /oss-only %}}
+will replicate all writes to the specified bucket to the {{% oss-only %}}remote {{% /oss-only %}}
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:213]:
```suggestion
will replicate all writes from the specified local bucket to the {{% oss-only %}}remote {{% /oss-only %}}
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 8]
```diff
@@ -85,6 +86,161 @@ InfluxDB {{% cloud-only %}}Cloud {{% /cloud-only %}}bucket.
...
+InfluxDB {{% cloud-only %}}Cloud {{% /cloud-only %}}bucket.
+Use the `/api/v2/replications` endpoint with the `GET` request method to return
+information such as the current queue size, max queue size, and latest status
+code for each replication stream.
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:217]:
```suggestion
To get
information such as the current queue size, max queue size, and latest status
code for each replication stream, send a `GET` request to your {{% oss-only %}}local{{% /oss-only %}} InfluxDB  OSS `/api/v2/replications` endpoint.
```

**Review by jstirnaman — COMMENTED** [review]
A few suggestions.

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 9]
```diff
@@ -85,6 +86,159 @@ InfluxDB {{% cloud-only %}}Cloud {{% /cloud-only %}}bucket.
...
+    {{< keep-url >}}
+    <a href="/influxdb/v2.4/api/#operation/PostRemoteConnection">
+    {{< api-endpoint endpoint="localhost:8086/api/v2/remotes" method="POST" >}}
+    </a>
+    
+    Include the following in your request:
+
+    - **Request method:** `POST`
+    - **Headers:**
+      - **Authorization:** `Token` schema with your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS [API token](/influxdb/v2.4/security/tokens/)
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:173]:
```suggestion
      - **Authorization:** `Token` scheme with your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS [API token](/influxdb/v2.4/security/tokens/)
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 10]
```diff
@@ -85,6 +86,159 @@ InfluxDB {{% cloud-only %}}Cloud {{% /cloud-only %}}bucket.
...
+    {{< keep-url >}}
+    <a href="/influxdb/v2.4/api/#operation/GetRemoteConnections">
+    {{< api-endpoint endpoint="localhost:8086/api/v2/remotes" method="GET" >}}
+    </a>
+
+    Include the following in your request:
+
+    - **Request method:** `GET`
+    - **Headers:**
+      - **Authorization:** `Token` schema with your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS [API token](/influxdb/v2.4/security/tokens/)
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:150]:
```suggestion
      - **Authorization:** `Token` scheme with your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS [API token](/influxdb/v2.4/security/tokens/)
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 11]
```diff
@@ -85,6 +86,159 @@ InfluxDB {{% cloud-only %}}Cloud {{% /cloud-only %}}bucket.
...
+    {{< keep-url >}}
+    <a href="/influxdb/v2.4/api/#operation/PostRemoteConnection">
+    {{< api-endpoint endpoint="localhost:8086/api/v2/remotes" method="POST" >}}
+    </a>
+
+    Include the following in your request:  
+
+    - **Request method:** `POST`
+    - **Headers:**
+      - **Authorization:** `Token` schema with your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS [API token](/influxdb/v2.4/security/tokens/)
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:107]:
```suggestion
      - **Authorization:** `Token` scheme with your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS [API token](/influxdb/v2.4/security/tokens/)
```

File: content/influxdb/v2.4/write-data/replication/replicate-data.md [hunk 12]
```diff
@@ -85,6 +86,159 @@ InfluxDB {{% cloud-only %}}Cloud {{% /cloud-only %}}bucket.
...
+{{< keep-url >}}
+<a href="/influxdb/v2.4/api/#operation/GetReplications">
+{{< api-endpoint endpoint="localhost:8086/api/v2/replications" method="GET" >}}
+</a>
+
+Include the following in your request:
+
+- **Request method:** `GET`
+- **Headers:**
+  - **Authorization:** `Token` schema with your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS [API token](/influxdb/v2.4/security/tokens/)
```
Comment (jstirnaman) [inline content/influxdb/v2.4/write-data/replication/replicate-data.md:226]:
```suggestion
  - **Authorization:** `Token` scheme with your {{% oss-only %}}local{{% /oss-only %}} InfluxDB OSS [API token](/influxdb/v2.4/security/tokens/)
```

**Review by jstirnaman — APPROVED** [review]
"scheme" > "schema"

**Review by jstirnaman — APPROVED** [review]

**Pull request merged**